README.md

```diff
@@ -82,7 +96,19 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+< sentence > ::= "The"< subject >< predicate >"."
+
+< subject >::=< noun >|< adjective >< noun >
+
+< noun >::="dog"|"cat"
+
+< adjective >::="smelly"|"lazy"
+
+< predicate >::=< verb >|< verb >< adverb >
+
+< verb >::="ate"|"ran"
+
+< adverb >::="slowly"|"noisily"
```

Comment:
# -3

Unfortunately this grammar doesn't allow for repeating adjectives or optional adverbs.